Source/SymbolResolver_AddMustToFermentingBarrels.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Verse;
+using RimWorld;
+using RimWorld.BaseGen;
+
+namespace RwWineMod
+{
+  public class SymbolResolver_AddMustToFermentingBarrels : SymbolResolver
+  {
+    public override void Resolve(ResolveParams rp)
+    {
+      Map map = BaseGen.globalSettings.map;
+      SymbolResolver_AddMustToFermentingBarrels.barrels.Clear();
+      CellRect.CellRectIterator iterator = rp.rect.GetIterator();
+      while (!iterator.Done())
+      {
+        List<Thing> thingList = iterator.Current.GetThingList(map);
+        for (int i = 0; i < thingList.Count; i++)
+        {
+          Building_FermentingBarrel building_FermentingBarrel = thingList[i] as Building_FermentingBarrel;
+          if (building_FermentingBarrel != null && !SymbolResolver_AddMustToFermentingBarrels.barrels.Contains(building_FermentingBarrel))
+          {
+            SymbolResolver_AddMustToFermentingBarrels.barrels.Add(building_FermentingBarrel);
+          }
+        }
+        iterator.MoveNext();
+      }
+      float progress = Rand.Range(0.1f, 0.9f);
```

Comment:
See this seems to add random progress and must to the barrel, I'm not sure what it is really used for.